.github/ISSUE_TEMPLATE/source_request.yml

```
@@ -0,0 +1,55 @@
+name: 🌐 Source request
+description: Suggest a new source for Tachiyomi
+labels: [Source Request]
```

Same here, capital change
```suggestion
labels: [Source request]
```

---

does this need to edit the label as well?

---

Yes, a capital change is required